tools/webpack/shared.js

```diff
@@ -69,8 +69,12 @@ const stylesTransform = ( mode ) => ( content ) => {
 	return content;
 };
 
+const normalizeJoin = ( baseDir, path ) => join( baseDir, path ).replace( /\\/g, '/' );
```

Comment:
It looks familiar. It's the same pattern we used to fix tooling in Gutenberg on Windows 😄

Reply:
Turns out there's even an [npm for that](https://github.com/sindresorhus/slash) 😅 (But I don't think we need that, easy enough to do it ourselves.)

Reply:
One thing we could consider -- since we're replacing occurrences of [`path.join()`](https://nodejs.org/api/path.html#pathjoinpaths) with this: We could allow for an arbitrary number of arguments (rather than two), to be more congruent with `path.join`.

Something like

```suggestion
const normalizeJoin = ( ...paths ) => join( ...paths ).replace( /\\/g, '/' );
```

Reply:
I have made that change locally and confirmed that I get the exact same build results 👍